example/android/build.gradle

```diff
@@ -30,6 +30,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
This is new in newer Gradle versions, was automatically added by the Flutter tool migrators.